Java/ScreenshotTaker/screenshottaker.java

```diff
@@ -0,0 +1,35 @@
+/* 
+Java Program to take a screenshot of the screen
+*/
+import java.awt.image.BufferedImage; 
+import java.io.File; 
+import java.awt.Rectangle; 
+import java.awt.Toolkit; 
+import java.awt.Robot; 
+import javax.imageio.ImageIO; 
+
+public class screenshottaker { 
+    public static final long serialVersionUID = 1L; 
+    public static void main(String[] args) 
+    { 
+        try 
+        {
+            Thread.sleep(120); //sleeping the main thread for 120 ms
+            Robot r = new Robot(); 
+
+            //to save the screen shot to a specific location
+            String path = "D:// Screenshot.jpg";
```

Comment:
As far as I can understand, this line will generate all screenshot with screenshot.jpg as the name. Can we make the naming of screenshot variable? Just try adding the time screenshot was taken to the end of the file to make it variable if possible.

Reply:
Yea sure...rather than that we can ask the user to decide the name of the screenshot.